clang/lib/DPCT/AnalysisInfo.cpp

```diff
@@ -669,7 +669,7 @@ void DpctFileInfo::insertHeader(HeaderType Type) {
                           "<" + getCustomMainHelperFileName() +
                               "/ccl_utils.hpp>");
     case HT_Atomic:
-      return insertHeader(HeaderType::HT_CCL, LastIncludeOffset,
+      return insertHeader(HeaderType::HT_Atomic, LastIncludeOffset,
```

Comment:
Please summit a seperate PR for this fix.

Reply:
done